wagtail_localize/test/migrations/0021_testuuidmodel_testuuidsnippet.py

```diff
@@ -0,0 +1,72 @@
+# Generated by Django 3.2.5 on 2022-03-17 20:20
+
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("wagtailcore", "0062_comment_models_and_pagesubscription"),
```

Comment:
I'd say drop this line as it is the source of test failures

Reply:
thanks.. removed it.. let's hope it fixed things 🤞